### PR TITLE
Save consensus leader after view change

### PR DESCRIPTION
**Pull request by ShengguangXiao**
## Description

Save consensus leader after view change, fix if recover after view change, the DS leader verifiy fail problem.

https://github.com/Zilliqa/Issues/issues/582

## Backward Compatibility

- [x] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [x] small-scale cloud test

### CODE REVIEW

File: src/libDirectoryService/ViewChangePostProcessing.cpp [hunk 1]
```diff
@@ -287,6 +287,12 @@ void DirectoryService::ProcessViewChangeConsensusWhenDone() {
     return;
   }
 
+  if (!BlockStorage::GetBlockStorage().PutDSCommittee(m_mediator.m_DSCommittee,
```
Comment (sandipbhoir) [inline src/libDirectoryService/ViewChangePostProcessing.cpp]: I think also needs to update the same on Lookup's after VC?

Comment (ShengguangXiao) [reply src/libDirectoryService/ViewChangePostProcessing.cpp]: Lookup and shard node will call function Node::ProcessVCDSBlocksMessage to process VCDS block, and the PutDSCommittee function is called inside it already.

Comment (ShengguangXiao) [reply src/libDirectoryService/ViewChangePostProcessing.cpp]: Fixed, please review again. @sandipbhoir

**Comment by ShengguangXiao** [comment]
Test with recovered community testnet, kill DS leader to trigger view change, after view change success and run for some blocks, to backup persistence before next DS block, and recover another testnet from the persistence, the retrieved the DS leader is the leader after view change.

**Pull request merged**